src/workflows/steps/conflictResolution.js

```diff
@@ -0,0 +1,155 @@
+import git from "../../git";
+import util from "../../utils";
+import logger from "better-console";
+import chalk from "chalk";
+import { remove } from "lodash";
+
+export function getLocalChanges( state ) {
+	const { dependencies } = state;
+
+	// creates an object of the changes you made to the package.json
+	// for the pre-releases.
+	const localChanges = dependencies.reduce( ( result, dep ) => {
+		result[ dep.pkg ] = "";
```

Comment:
Unless I'm thinking incorrectly, I don't think we need to assign twice. You should be able to delete this line and let the next line do it's thang.